### PR TITLE
Adding PEM certificate troubleshooting

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Adding Zowe CLI troubleshooting content for PEM certificates

List the file(s) included in this PR: troubleshoot-cli-credentials.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/troubleshoot/cli/troubleshoot-cli-credentials.md [hunk 1]
```diff
@@ -11,3 +15,42 @@ Zowe CLI accepts various methods, or mechanisms, of authentication when communic
 However, some services can accept multiple methods of authentication. When multiple methods are provided (in a profile or command) for a service, the CLI follows an *order of precedence* to determine which method to apply.
 
 To find the authentication methods used for different services and their order of precedence, see the table in [Authentication mechanisms](../../extend/extend-cli/cli-devTutorials.md#authentication-mechanisms).
+
+### PEM certificate files
+
+PEM certificate files are one of the authentication mechanisms used by Zowe CLI.
```
Comment (gejohnston) [inline docs/troubleshoot/cli/troubleshoot-cli-credentials.md:21]: Should we state that a client certificate must be recorded in the mainframe service's trust store or keyring before the client certificate will be accepted by the mainframe service?

Should we state that the Zowe client certificate is a separate certificate from the certificate used by the APIML gateway to authenticate to the APIML service?

**Review by gejohnston — APPROVED** [review]
I suggested a couple of small possible additions.
I will am willing to defer to team consensus on whether or not to include those suggestions.

File: docs/troubleshoot/cli/troubleshoot-cli-credentials.md (hunk 1 above)
Comment (anaxceron) [reply docs/troubleshoot/cli/troubleshoot-cli-credentials.md]: updated the content accordingly -- thank you!

**Review by awharn — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
Thank you for addressing the comments 😋 
LGTM! 😋

**Comment by sbohnert** [comment]
Looks good, Ana. Thanks

**Review by adam-wolfe — APPROVED** [review]

**Pull request merged**